.github/workflows/CD.yml

```diff
@@ -1,8 +1,9 @@
 name: CD
 
 on:
```

Comment:
Won't empty `on` section trigger it on any activity?

Reply:
`workflow_dispatch` here